flox/core.py

```diff
@@ -35,6 +36,17 @@
 if TYPE_CHECKING:
     import dask.array.Array as DaskArray
 
+    T_Func = Union[str, Callable]
+    T_Funcs = Union[T_Func, Sequence[T_Func]]
+    T_Axis = int
+    T_Axiss = tuple[T_Axis, ...]  # TODO: var name grammar?
```

Comment:
```suggestion
    T_Axes = tuple[T_Axis, ...]  # TODO: var name grammar?
```

prefer this spelling

Reply:
Done